### PR TITLE
edited publications

### PR DESCRIPTION
**Pull request by lena-hinrichsen**
still need javascript to display the publications properly

### CODE REVIEW

**Comment by kba** [comment]
@lena-hinrichsen I added the javascript that turns the "Aufzeichnung" into links if they are links or hides them if they are not. Please try it out to make sure I didn't mess up anything with the delimiters etc. Thanks!

**Review by kba — APPROVED** [review]

**Comment by lena-hinrichsen** [comment]
looks good! But I could change the Zotero style so it says "Video" instead of "Aufzeichnung" so it works in both languages

**Comment by kba** [comment]
> looks good! But I could change the Zotero style so it says "Video" instead of "Aufzeichnung" so it works in both languages

Good point, yes let's do "Video" instead of "Aufzeichnung". Can you add the updated CSL to the PR? I'll update the JS accordingly.

**Comment by lena-hinrichsen** [comment]
> Can you add the updated CSL to the PR? I'll update the JS accordingly.

I don't know why but the script does not work anymore for me ...

File: zotero-ocrd-infoclioch.csl [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+                    <text variable="URL" prefix="&lt;" suffix="&gt;"/>
+                    <text macro="accessed"/>
+                </group>
+            </else-if>
+        </choose>
+        <choose>
+            <if variable="title-short">
+                <text value=" "/>
+                <group delimiter=": ">
+                    <text value="; Video"/>
```
Comment (kba) [inline zotero-ocrd-infoclioch.csl:457]: The `; ` in `; Video` seems redundant, produces to consecutive `;`

**Pull request merged**